stdlib/public/core/SIMDVector.swift

```diff
@@ -4,13 +4,6 @@ infix operator .< : ComparisonPrecedence
 infix operator .<= : ComparisonPrecedence
 infix operator .> : ComparisonPrecedence
 infix operator .>= : ComparisonPrecedence
-infix operator .& : LogicalConjunctionPrecedence
-infix operator .^ : LogicalDisjunctionPrecedence
-infix operator .| : LogicalDisjunctionPrecedence
-infix operator .&= : AssignmentPrecedence
-infix operator .^= : AssignmentPrecedence
-infix operator .|= : AssignmentPrecedence
-prefix operator .!
```

Comment:
These should probably still be declared in the stdlib even if there are no functions that implement them.